Mathlib/MeasureTheory/Measure/LevyProkhorovMetric.lean

```diff
@@ -219,6 +224,196 @@ noncomputable instance levyProkhorovDist_pseudoMetricSpace_probabilityMeasure :
   dist_triangle μ ν κ := levyProkhorovDist_triangle _ _ _
   edist_dist μ ν := by simp [← ENNReal.ofReal_coe_nnreal]
 
+lemma levyProkhorov_dist_def (μ ν : LevyProkhorov (ProbabilityMeasure Ω)) :
+    dist μ ν = levyProkhorovDist μ.toMeasure ν.toMeasure := rfl
+
 end Levy_Prokhorov --section
 
+section Levy_Prokhorov_comparison
+
+open BoundedContinuousFunction
+
+variable {ι : Type*} (Ω : Type*) [MeasurableSpace Ω]
+
+/-- Coercion from the type synonym `LevyProkhorov (ProbabilityMeasure Ω)`
+to `ProbabilityMeasure Ω`. -/
+def levyProkhorov_to_probabilityMeasure (μ : LevyProkhorov (ProbabilityMeasure Ω)) :
```

Comment:
I would probably name this `LevyProkhorov.probabilityMeasure` to be able to use dot notation `μ.probabilityMeasure`. With Ω implicit: if you want to use the function without variable, just use `LevyProkhorov.probabilityMeasure (Ω := Ω)`.